osmtm/templates/start.mako

```diff
@@ -10,17 +10,17 @@ login_url= request.route_path('login', _query=[('came_from', request.url)])
 <div class="container">
   <div class="row">
     <h2>
-      Welcome, new user!
+      ${_('Welcome, new user!')}
     </h2>
     <div>
       <p>
-        You're ready to use the Tasking Manager.
+        ${_('You\'re ready to use the Tasking Manager.')}
       </p>
       <p>
-        Please <a href="${login_url}">login</a>.
+        ${_('Please')} <a href="${login_url}">${_('login')}</a>
```

Comment:
It would be better if this sentence is not splitted into two.
"please login" is probably translated differently than "please" and "login".